content/applications/sales/crm/acquire_leads/lead_mining.rst

```diff
@@ -1,71 +1,75 @@
 =======================
 Lead mining
 =======================
-In any business, getting quality leads is essential to keep the business growing. Lead mining allows you to generate leads from scratch directly from your database.
-Target your leads based on a set of criteria such as the country, the size of the company, the industry your leads are coming from to make them relevant to your business.
+In any business, getting quality leads is essential to keep the business growing. Lead mining
+allows you to generate leads from scratch directly within your database. Leads can be targeted based
+upon a set of criteria, such as the country, the company size, and the industry, to ensure the
+leads are relevant to your business.
 
 Configuration
 ==============
 
-For this feature to work, go to :menuselection:`CRM --> Configuration --> Settings` and activate the **Lead Mining** feature.
+For this feature to work, go to :menuselection:`CRM --> Configuration --> Settings` and activate
+the **Lead Mining** feature.
  
-.. image:: media/LM1.png
+.. image:: lead_mining/LM1.png
    :align: center
-
+   :alt: Activating Lead Mining
 
 Start generating leads
 ==========================
-You will now have a new button **Generate Leads** available in your pipeline.
-You are also able to create lead mining requests from the
-:menuselection:`Configuration --> Lead Mining Requests` and trough
+You will now have a new button **Generate Leads** available in your pipeline. Lead mining
+requests are also available from :menuselection:`Configuration --> Lead Mining Requests` or through
 :menuselection:`Leads --> Leads` where you have the **Generate Leads** button.
 
-.. image:: media/LM2.png
+.. image:: lead_mining/generate-leads-button.png
    :align: center
+   :alt: Generate Leads Buttons
 
+From there, click on the **Generate Leads** button, a window where you will be able to pick your
+criteria will pop up.
 
-From there, click on the **Generate Leads** button, a window where you will be able to pick your criteria will pop up.
-
-
-.. image:: media/LM5.png
+.. image:: lead_mining/generate-leads-popup.png
    :align: center
+   :alt: Generating Leads Pop-Up
```

Comment:
Wouldn't this one have a capitalization on the "Leads" since in the app itself "Generate Leads" is capitalized? I think in this case, it makes more sense to rename the alt text to "Generate Leads pop-up." Thoughts?